.gitignore

```diff
@@ -44,3 +44,7 @@ docker/pgdata/
 next-env.d.ts
 
 apps/web/public/uploads
+
+# Misc
+TODO.md
+.cursorrules
```

Comment:
I'm giving it a try given all the hype

let me know if u wanna a license to try it